engine-sdk/Cargo.toml

```diff
@@ -14,7 +14,7 @@ autobenches = false
 
 [dependencies]
 aurora-engine-types = { path = "../engine-types", default-features = false }
-borsh = { version = "0.8.2", default-features = false }
+borsh = { version = "0.9", default-features = false }
```

Comment:
To prevent potential issues, can you please peg this to 0.9.3 in all Cargo.toml files?

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/560/commits/2d91055d16d54651b1ca143f34ccd6ce1921d856